iphone/Classes/TiUIProgressBar.m

```diff
@@ -21,13 +21,13 @@ -(void)initializeTiLayoutView
 }
 #endif
 
--(id)initWithStyle:(UIProgressViewStyle)style_
+-(id)initWithStyle:(UIProgressViewStyle)_style andMinimumValue:(CGFloat)_min maximumValue:(CGFloat)_max;
 {
 	if (self = [super initWithFrame:CGRectZero])
 	{
-		style = style_;
-		min = 0;
-		max = 1;
+		style = _style;
+		min = _min;
+		max = _max;
```

Comment:
I know but we have setter methods for min and max. Not for style , i guess doing that much for it would be redundant however so never mind.